src/libs/ErrorUtils.js

```diff
@@ -88,6 +88,26 @@ function getLatestErrorField(onyxData, fieldName) {
         .value();
 }
 
+/**
+ * @param {Object} onyxData
+ * @param {Object} onyxData.errorFields
+ * @param {String} fieldName
+ * @returns {Object}
+ */
+function getFirstErrorField(onyxData, fieldName) {
```

Comment:
It looks like the only difference between this function and `getLatestErrorField` is that `getLatestErrorField` adds a `.reverse()` call in the chain... I think I'd prefer we either:
1. Rename this to something like `getEarliestErrorField` (so we know it's "first" by **timestamp**)
2. Update `getLatestErrorField` with a flag to allow us to get earliest OR latest in the same function

Thoughts?

Reply:
@hoangzinh i think i'm fine with that for now 👍  can you please make that quick change then I can merge this PR? 🙏